src/test/scala/org/hammerlab/guacamole/readsets/rdd/CoverageRDDSuite.scala

```diff
@@ -26,88 +24,41 @@ class CoverageRDDSuiteRegistrar extends KryoTestRegistrar {
   }
 }
 
-class RegionRDDSuite
+class CoverageRDDSuite
   extends GuacFunSuite
     with RegionsRDDUtil
     with ContigLengthsUtil {
 
   override def registrar = "org.hammerlab.guacamole.readsets.rdd.CoverageRDDSuiteRegistrar"
```

Comment:
Doesn't matter much here since this class is defined in this file, but you can do `classOf[CoverageRDDSuiteRegistrar].getCanonicalName` to protect against re-namings or packages moving around.